tsl/src/chunk_copy.c

```diff
@@ -635,21 +643,27 @@ chunk_copy_stage_sync(ChunkCopy *cc)
 static void
 chunk_copy_stage_drop_subscription(ChunkCopy *cc)
 {
-	char *cmd;
+	char *cmd1, *cmd;
 
 	/* Stop data transfer on the destination node */
-	cmd = psprintf("ALTER SUBSCRIPTION %s DISABLE", NameStr(cc->fd.operation_id));
+	cmd1 = psprintf("ALTER SUBSCRIPTION %s DISABLE", NameStr(cc->fd.operation_id));
+	cmd = psprintf("SELECT timescaledb_experimental.subscription_cmd($sql$%s$sql$)", cmd1);
```

Comment:
Since those command are repeated, I am suggesting to create a function which executes it remote as a wrapper:
```

static void
chunk_copy_execute_subscription_command(ChunkCopy *cc, const char *command, data nodes ...)
{
// include command in the argument to subscription_cmd()
// execute on data nodes
}

or

static void
chunk_copy_subscription_exec(...)
{
}
```

Reply:
+1 for @pmwkaa suggestion

Reply:
@pmwkaa @fabriziomello done the above